src/util/fieldHelpers.js

```diff
@@ -3,6 +3,8 @@
 // In the codebase, we also have React Final Form fields, which are wrapper around user inputs.
 
 import { SCHEMA_TYPE_MULTI_ENUM, SCHEMA_TYPE_TEXT, SCHEMA_TYPE_YOUTUBE } from './types';
+import { BOOKING_PROCESS_NAME, PURCHASE_PROCESS_NAME } from '../transactions/transaction';
+import { stripeSupportedCurrencies, subUnitDivisors } from '../config/settingsCurrency.js';
```

Comment:
Note: util/types.js loads transaction.js file. Therefore, it would be better if transaction.js is loaded before types. settingsCurrency.js does not import anything, but don't call it directly. use settings instead.


```suggestion
import appSettings from '../config/settings';
// import { BOOKING_PROCESS_NAME, PURCHASE_PROCESS_NAME } from '../transactions/transaction';
import { isPurchaseProcessAlias, isBookingProcessAlias } from '../transactions/transaction';
import { SCHEMA_TYPE_MULTI_ENUM, SCHEMA_TYPE_TEXT, SCHEMA_TYPE_YOUTUBE } from '../util/types';

const { stripeSupportedCurrencies, subUnitDivisors } = appSettings;
```

Note: you need to modify **settings.js** too.